apps/core/components/sectorDetail/SectorDetail.js

```diff
@@ -12,11 +14,29 @@ type Props = {|
 |};
 
 const SectorDetail = (props: Props) => {
+  const segmentsData = props.data?.segments ?? [];
+  const segments = segmentsData.map((data, index) => {
+    if (data) {
+      return (
+        <React.Fragment key={data.id}>
+          <Segment data={data} />
+          {index < segmentsData.length - 1 && (
```

Comment:
Maybe this could be separated in some const, it could be more readable, what do you think?

Reply:
👍 separated